spark/src/main/scala/com/alibaba/graphar/EdgeInfo.scala

```diff
@@ -282,10 +282,51 @@ class EdgeInfo() {
     return true
   }
 
+  /** Get the vertex num file path
+   *
+   *  @param adj_list_type type of adj list structure.
+   *  @return the vertex num file path. If edge info not support the adj list type,
+   *          raise an IllegalArgumentException error.
+   */
+  def getVerticesNumFilePath(adj_list_type: AdjListType.Value): String = {
+    if (containAdjList(adj_list_type) == false)
+      throw new IllegalArgumentException
+    val str: String = prefix + getAdjListPrefix(adj_list_type) + "vertex_count"
+    return str
+  }
+
+  /** Get the path prefix of the edge num file path
+   *
+   *  @param adj_list_type type of adj list structure.
+   *  @return the edge num file path. If edge info not support the adj list type,
+   *          raise an IllegalArgumentException error.
+   */
+  def getEdgesNumPathPrefix(adj_list_type: AdjListType.Value): String = {
+    if (containAdjList(adj_list_type) == false)
+      throw new IllegalArgumentException
+    val str: String = prefix + getAdjListPrefix(adj_list_type) + "edge_count"
+    return str
+  }
```

Comment:
I am a little  puzzled when we use this function if we already have function `getEdgesNumFilePath(chunk_index, adj_list_type)`

Reply:
This function is used for getting the total number of edges without getting the vertex chunk number at first.

Reply:
Did we have a `edge_count` file to store the total edge num?

Reply:
No, you can refer to the [test data](https://github.com/GraphScope/gar-test/tree/main/ldbc_sample/csv/edge/person_knows_person/ordered_by_source). But the total edge num can be got by summing up.

Reply:
I'm not get this, if we don't have a `edge_count` file, how can we use this function to get the total number of edges, could you show me some code example?

Reply:
val edge_num_path_pattern = new Path(prefix + edge_info.getEdgesNumPathPrefix(adj_list_type) + "*")
    val edge_num_files = fs.globStatus(edge_num_path_pattern)
    val edge_num = edge_num_files.map(file => utils.FileSystem.readValue(file.getPath().toString(), spark.sparkContext.hadoopConfiguration).toInt).sum

Reply:
> ```
> val edge_num_path_pattern = new Path(prefix + edge_info.getEdgesNumPathPrefix(adj_list_type) + "*")
> val edge_num_files = fs.globStatus(edge_num_path_pattern)
> val edge_num = edge_num_files.map(file => utils.FileSystem.readValue(file.getPath().toString(), spark.sparkContext.hadoopConfiguration).toInt).sum
> ```

Great, this looks reasonable to me now.